wallet/src/de/schildbach/wallet/ui/dashpay/notification/TransactionViewHolder.kt

```diff
@@ -42,8 +41,8 @@ import org.dash.wallet.common.ui.CurrencyTextView
 import org.dash.wallet.common.util.GenericUtils
 import java.util.*
 
-class TransactionViewHolder(inflater: LayoutInflater, parent: ViewGroup) :
-        NotificationViewHolder(R.layout.notification_transaction_row, inflater, parent) {
+class TransactionViewHolder(val binding: NotificationTransactionRowBinding) :
+        NotificationViewHolder(binding.root) {
```

Comment:
This class uses `findViewById` but these were not updated to use `binding`.  This can be done with a different story.